core/src/main/java/org/apache/gravitino/storage/relational/po/PolicyPO.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.storage.relational.po;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import lombok.Getter;
+
+@Getter
+public class PolicyPO {
+  private Long policyId;
+  private String policyName;
+  private String policyType;
+  private Long metalakeId;
+  private boolean inheritable;
+  private boolean exclusive;
+  private String supportedObjectTypes;
+  private String auditInfo;
+  private Long currentVersion;
+  private Long lastVersion;
+  private Long deletedAt;
+  private PolicyVersionPO policyVersionPO;
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof PolicyPO)) {
+      return false;
+    }
+    PolicyPO policyPO = (PolicyPO) o;
+    return Objects.equal(getPolicyId(), policyPO.policyId)
+        && Objects.equal(getPolicyName(), policyPO.policyName)
+        && Objects.equal(getPolicyType(), policyPO.policyType)
+        && Objects.equal(getMetalakeId(), policyPO.metalakeId)
+        && Objects.equal(isInheritable(), policyPO.inheritable)
+        && Objects.equal(isExclusive(), policyPO.exclusive)
+        && Objects.equal(getSupportedObjectTypes(), policyPO.supportedObjectTypes)
+        && Objects.equal(getAuditInfo(), policyPO.auditInfo)
+        && Objects.equal(getCurrentVersion(), policyPO.currentVersion)
+        && Objects.equal(getLastVersion(), policyPO.lastVersion)
+        && Objects.equal(getPolicyVersionPO(), policyPO.policyVersionPO)
+        && Objects.equal(getDeletedAt(), policyPO.deletedAt);
```

Comment:
Shall we call the method or the variable, I think we should be consistent with other implementation.